sandbox/TOC.yml

```diff
@@ -41,10 +41,11 @@
       expanded: true
       href: functions-recipes/index.yml
       items:
-      - name: Functions basics
+      - name: Functions Basics
```

Comment:
Every other docs PR I've seen requires sentence casing - e.g. "Functions basics" instead of "Functions Basics". Do these docs go by a different standard? Same thing for section headers.